static/src/stylesheets/layout/new-header/_header-cta-item.scss

```diff
@@ -77,10 +76,10 @@
 
     &:hover,
     &:focus {
-        color: currentColor;
+        color: $guardian-brand-dark;
```

Comment:
Tried this and it feels a little too extreme. Will review when I look at accessibility changes from UX though 👍